.github/workflows/coverage.yml

```diff
@@ -0,0 +1,73 @@
+# Syntax reference:
+# https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions
+
+name: Coverage
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  push:
+    branches: [dev]
+  pull_request:
+    types: [opened, synchronize]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  codecov:
+    runs-on: ubuntu-latest
```

Comment:
Just a thought - I think ubuntu 20 is available "natively" now, and is actually the new default? So hypothetically we could specify 20 here and not use a container? Otoh using a container doesn't add much overhead afaik, and would keep it closer to the other workflows.

Reply:
I think the coverage test requires 20.10 for clang 11, but GitHub only has LTS releases: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources